tests/data/conf_covariance_cNG_fsky.yaml

```diff
@@ -0,0 +1,68 @@
+tjpcov:
+  # sacc input file 
+  sacc_file: ./tests/benchmarks/32_DES_tjpcov_bm/cls_cov.fits
+
+  # 'set' from parameters OR pass CCL cosmology object OR yaml file
+  cosmo: 'set'
+
+  # Setting mask OR fsky approximation
+  mask_file:
```

Comment:
`masks` should not be required when using `fsky`. Remove them and check that nothing breaks (I think we already tested this for the SSC)